### PR TITLE
FDP-842: CycloneDX SBOM plugin configured

### PR DESCRIPTION
**Pull request by sanderv**
- Enabled for each .war project
- bom.* is packaged in each .war for Openshift VI scanning

### CODE REVIEW

File: super/pom.xml [hunk 1]
```diff
@@ -1157,6 +1161,51 @@
...
+      <id>spring-roo-repository</id>
+      <url>https://repo1.maven.org/maven2/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+-->
```
Comment (smvdheijden) [inline super/pom.xml]: Shouldn't this block be removed?

File: super/pom.xml [hunk 2]
```diff
@@ -472,6 +472,10 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-aop</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>com.querydsl</groupId>
+            <artifactId>querydsl-apt</artifactId>
+          </exclusion>
```
Comment (smvdheijden) [inline super/pom.xml]: Is this exclusion needed for getting the CycloneDC SBOM plugin to work?

File: super/pom.xml (hunk 1 above)
Comment (sanderv) [reply super/pom.xml]: removed

File: super/pom.xml (hunk 2 above)
Comment (sanderv) [reply super/pom.xml]: removed

**Review by smvdheijden — APPROVED** [review]

**Pull request merged**